ambari/datadog_checks/ambari/ambari.py

```diff
@@ -49,12 +49,17 @@ def _should_collect_service_status(self):
     def get_clusters(self):
         clusters_endpoint = common.CLUSTERS_URL.format(base_url=self.base_url)
         service_check_tags = self.base_tags + ["url:{}".format(self.base_url)]
-        resp = self._make_request(clusters_endpoint)
-        if resp is None:
+
+        try:
```

Comment:
I'm not completely sure if this is the best implementation, but the original implementation would throw an error with a bad instance/couldn't connect, and it wouldn't emit a service check.